stumpy/cache.py

```diff
@@ -107,3 +109,40 @@ def _get_cache():
     site_pkg_dir = site.getsitepackages()[0]
     numba_cache_dir = site_pkg_dir + "/stumpy/__pycache__"
     return [f.name for f in pathlib.Path(numba_cache_dir).glob("*nb*") if f.is_file()]
+
+
+def _recompile(func=None, fastmath=None):
```

Comment:
@seanlaw 
I did not add support for clearing cache to this function. Not sure how that fits into the picture here.